fn_falcon_sandbox/fn_falcon_sandbox/components/falcon_sandbox_submit_file.py

```diff
@@ -39,6 +42,7 @@ def __init__(self, opts):
         """constructor provides access to the configuration options"""
         super(FunctionComponent, self).__init__(opts)
         self.options = opts.get("fn_falcon_sandbox", {})
+        selftest.selftest_function(opts)
```

Comment:
* No need to import selftest here. This means it will be ran each time the function is invoked
* To run selftest we would use the command:
```
$ resilient-circuits selftest -l fn-falcon-sandbox
```